persistence/sql/persister_continuity.go

```diff
@@ -41,3 +42,18 @@ func (p *Persister) DeleteContinuitySession(ctx context.Context, id uuid.UUID) e
 	}
 	return nil
 }
+
+func (p *Persister) DeleteExpiredContinuitySessions(ctx context.Context, expiresAt time.Time, limit int) error {
+	// #nosec G201
+	err := p.GetConnection(ctx).RawQuery(fmt.Sprintf(
+		"DELETE FROM %s WHERE expires_at <= ? LIMIT ?",
```

Comment:
Please see https://github.com/gobuffalo/pop/pull/658 and specifically https://github.com/gobuffalo/pop/pull/658/files#diff-5d204b98ec636a0fe75d32bf558fc3c06ff540eb1010ede6144e788c0c3aa57aR131-R140

Reply:
this will be fun. Removing limits in our case will be tricky (or we'll just cleanup with our version where we use limits https://github.com/Wikia/kratos/pull/56 and then just run this versionlater on ). As long as we run it more frequently this shouldn't be an issue

Reply:
> This will be fun.

Is limit-less deletion for an established system a big performance impact? If so, we should probably add some kind of limit, e.g. using a sub-query, that works on all DBs.

Reply:
Yeah this is tricky since we didn't have this ability from the start and we can't remove everything all at once, even with using the dates like in the pr. When switching from one system to another we created a lot of sessions/flows etc at the same moment time.  Dropping those records can cause a big replication lag when the delete query hits that moment in time. Using limit helps us mitigate the problem a little and drops them in batches. This way we don't have to run this jub as often and there is no need to monitor this as much